cpp/arcticdb/column_store/chunked_buffer.hpp

```diff
@@ -271,7 +271,11 @@ class ChunkedBufferImpl {
     }
 
     [[nodiscard]] const uint8_t* data() const {
-        util::check(blocks_.size() == 1, "Taking a pointer to the beginning of a non-contiguous buffer");
+        if (blocks_.empty()) {
+            return nullptr;
+        }
+        internal::check<ErrorCode::E_ASSERTION_FAILURE>(blocks_.size() == 1,
```

Comment:
Do we still need this?

Reply:
Yes, if there are multiple blocks this will not work as expected